pp/integrations/plugin-packs/procedures/applications-cisco-ssms-restapi.md

```diff
@@ -50,102 +64,319 @@ a way to access a subset of Cisco SSM functionality without using a direct Inter
 A service account is required to query the Cisco SSMS API. It must have sufficient read privileges in the account targeted.
```

Comment:
2nd sentence sounds weird, what is it trying to say?

Reply:
A service account with read-only privileges is required to query the Cisco SSMS API.